test/messages-test.js

```diff
@@ -1,40 +1,48 @@
+import Gists from 'gists'
+import GoogleSpreadsheet from 'google-spreadsheet'
 import async from 'async'
 import chai from 'chai'
 import sinon from 'sinon'
 let messages = require('../scripts/messages.js').messages
 
 
 describe('Google Docs and Gists Logging', function () {
-    it('Google Docs messaging', function () {
-        let message = {
-            "text": "+todo testing @test",
-            "user": {
-                "name": "test"
-            }
-        }
-        sinon.stub(messages, "sendMessage", function () {
-            return "0"
+    let save
+
+    beforeEach(function () {
+        save = null
+    })
+
+    it('Formatting message', function () {
+        save = sinon.stub(messages, 'getRoom').callsFake(function (roomId, callback) {
+            sinon.assert.calledOnce(save)
+            callback("testname")
         })
 
-        messages.sendMessage(message, "dest", function (inf) {})
+        messages.formatMessage({"text": "+todo @test this", "room": "testingName", "user": {"name": "test", "login": "test"}}, function (data) {
+        })
+        save.reset()
+    })
 
-        sinon.assert.calledOnce(messages.sendMessage)
+    it('GDoc Logging', function () {
+        save = sinon.stub(messages, 'formatMessage').callsFake(function (message, callback) {
+            sinon.assert.calledOnce(save)
+        })
+        messages.sendMessage({"text": "+todo @test this", "user":{"name": "test", "login": "test"}}, "gdoc", function (data) {
 
+        })
+        save.reset()
     })
 
-    it('Gist messaging', function () {
-        let message = {
-            "text": "+todo testing @test",
-            "user": {
-                "name": "test"
-            }
-        }
-        sinon.stub(messages, "sendGist", function () {
-            return "0"
+    it('Gists Logging', function () {
+        save = sinon.stub(new Gists(), 'download').callsFake(function (dest, callback) {
+            sinon.assert.calledOnce(save)
         })
 
-        messages.sendGist(message, "dest", function (inf) {})
+        messages.sendGist({"text": "+todo @test this", "user":{"name": "test", "login": "test"}}, "gdoc", function (data) {
 
-        sinon.assert.calledOnce(messages.sendGist)
+        })
+        save.reset()
```

Comment:
Again all this 3 test cases are useless. They are not actually testing anything. If you remove any of this calls from test cases it still passes
```
it('Gists Logging', function () {
        save = sinon.stub(new Gists(), 'download').callsFake(function (dest, callback) {
            sinon.assert.calledOnce(save)
        })
        save.reset()
    })
``` 
Eg if you add example code above to tests, it still passes